lib/_cookieUtils.ts

```diff
@@ -39,29 +40,19 @@ const getCookie = (cname: string): string => {
  * Return new UUID
  * @return {[string]} new UUID
  */
-const checkUserIdCookie = (userSpecifiedID?: string | number): string => {
+const checkUserIdCookie = (
+  userSpecifiedID?: string | number,
+  cookieDuration?: number
+): string => {
   const userToken = getCookie(COOKIE_KEY);
 
   if (!userToken || userToken === "") {
     const newUUID = createUUID();
-    setCookie(COOKIE_KEY, newUUID, 10);
+    setCookie(COOKIE_KEY, newUUID, cookieDuration);
     return newUUID;
   }
-  setCookie(COOKIE_KEY, userToken, 10);
-  return userToken;
-};
 
-/**
```

Comment:
I moved createUUID outside so I can mock it in the tests. It's generic enough that it can belong inside utils/index.ts